### PR TITLE
Add implementation to ui.dart and a web project for testing it.

### PR DESCRIPTION
**Pull request by marianamarasoiu**
@lukechurch PTAL :)

### CODE REVIEW

File: vis/web/packages [hunk 1]
```diff
@@ -0,0 +1 @@
+../packages
```
Comment (marianamarasoiu) [inline vis/web/packages]: @lukechurch Do we commit packages files?

File: vis/lib/api.dart [hunk 2]
```diff
@@ -7,12 +7,13 @@ import 'ui.dart';
 int addLine(
   int startX, int startY,
   int endX, int endY,
+  int width,
```
Comment (lukechurch) [inline vis/lib/api.dart]: consider s/width/thickness

File: vis/lib/ui.dart [hunk 3]
```diff
@@ -1,6 +1,64 @@
...
+svg.SvgSvgElement _svgContainer;
+
+initUI(html.Element parent, int width, int height) {
+  _svgContainer = new svg.SvgSvgElement()
+    ..attributes['width'] = '$width'
+    ..attributes['height'] = '$height';
+  parent.append(_svgContainer);
+}
+
+Map<int, svg.SvgElement> cache = {};
```
Comment (lukechurch) [inline vis/lib/ui.dart]: It's not really a cache. It's an IdToElement map.

File: vis/web/packages (hunk 1 above)
Comment (lukechurch) [inline vis/web/packages]: Please don't add packages paths to the repo

**Review by lukechurch — APPROVED** [review]
Thanks for this.

Minor renaming and a path problem. Could you fix and then merge?

File: vis/web/packages (hunk 1 above)
Comment (lukechurch) [reply vis/web/packages]: Heh - see below :)

**Pull request merged**